### PR TITLE
Container for assuming role based on account numbers and env

### PR DESCRIPTION
**Pull request by stevenjack**
### Problem

Currently to assume a role we need to install the `aws-cli` and `jq`, this could be contained in a container so it's a one off command for the developer.

### Solution

Create a container for assuming a role.

### Notes

* https://trello.com/c/UgK84kOx/418-docker-images-assume-role-container-with-dependencies

### CODE REVIEW

File: assume-role/assume-role.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# assume-role <staging|live>
```
Comment (revett) [inline assume-role/assume-role.sh]: Update this

Comment (revett) [inline assume-role/assume-role.sh]: or remove

File: assume-role/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM python:slim
+MAINTAINER Steven Jack <smaj@vidsy.co>
+
+RUN pip install --upgrade pip
+RUN pip install awscli
```
Comment (revett) [inline assume-role/Dockerfile]: Install specific version of the CLI

File: assume-role/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+RUN apt-get update
```
Comment (revett) [inline assume-role/Dockerfile]: Remove cache after

**Comment by revett** [comment]
:cake: after comments

**Comment by revett** [comment]
:cake:

**Comment by archievidsy** [comment]
:cactus:

**Pull request merged**